src/views/UserScreen.tsx

```diff
@@ -12,6 +14,21 @@ export interface UserScreenProps {
 export default function UserScreen(props: UserScreenProps) {
   const { user } = props;
 
+  const { data, loading, refetch } = useQuery(ME_QUERY, {
+    notifyOnNetworkStatusChange: true,
+    context: {
```

Comment:
does this need to be defined here in addition to the more general case you seem to have set up when you created the apollo client in App.tsx?